action.yml

```diff
@@ -11,10 +11,15 @@ inputs:
   ignore_sha:
     description: 'Optional - Allow canceling other workflows with the same SHA. Useful for the `pull_request.closed` event.'
     required: false
-    default: false
+    default: 'false'
```

Comment:
Why was this changed from `false` to `'false'`?

Reply:
In [the specification](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault), `default` is a string value. In YAML true and false are booleans, so adding the `'`s turns this it into the string `"false"`.

The way YAML handles strings causes a number of common gotchas, especially when it comes to booleans.